README.md

```diff
@@ -1199,6 +1199,7 @@ Table of Contents
 
 ## Screenshot APIs
 
+  * [ProxyCrawl.com](https://proxycrawl.com/screenshots-api) - Automate website screenshots while avoiding captchas and blocks with an API. Free tier includes 50 screenshots (one time, not monthly) with all the available functionality.
```

Comment:
so its a one off 50 screenshots free? That would unfortunately not qualify for this list I am afraid. It just isnt a free tier, compare to the requirement where free services offered has to be for at least a year this just doesnt cut it.

If it was like 5 or 10 a month, great, as is afraid its not enough for this list